src/Development/IDE/Core/FileStore.hs

```diff
@@ -230,7 +230,7 @@ typecheckParents state nfp = void $ shakeEnqueue (shakeExtras state) parents
 
 typecheckParentsAction :: NormalizedFilePath -> Action ()
 typecheckParentsAction nfp = do
-    fs <- useNoFile_ GetKnownFiles
+    fs <- useNoFile_ GetKnownTargets
     unless (null fs) $ do
```

Comment:
If `KnownTargets` is always populated here, I don't think we need this check anymore.

Reply:
this one can be removed indeed